generate_clients.py

```diff
@@ -129,6 +109,10 @@ def parse_args(*args):
         # Default to the artman-specified default location.
         default="~/.artman/config.yaml",
         help='Specify where the artman user config lives.')
+    parser.add_argument(
+        '--languages',
```

Comment:
Can we use `nargs` here? https://docs.python.org/3.3/library/argparse.html#nargs